app/models/journeyDomain/incident/equipment/EquipmentDomain.scala

```diff
@@ -64,18 +66,20 @@ object EquipmentDomain {
       case false => UserAnswersReader.apply(ItemNumbersDomain(Nil))
     }
 
-    lazy val sealsReadsByIncidentCode = IncidentCodePage(incidentIndex).reader.flatMap {
-      case x if x.code == "2" => sealsReads
-      case _                  => optionalSealsReads
+    lazy val sealsReadsByIncidentCode: Kleisli[EitherType, UserAnswers, SealsDomain] = IncidentCodePage(incidentIndex).reader.flatMap {
+      _.code match {
+        case SealsBrokenOrTamperedCode => sealsReads
+        case _                         => optionalSealsReads
+      }
     }
 
-    lazy val readsWithContainerId = (
+    lazy val readsWithContainerId: Kleisli[EitherType, UserAnswers, EquipmentDomain] = (
       ContainerIdentificationNumberPage(incidentIndex, equipmentIndex).reader.map(Some(_)),
       sealsReadsByIncidentCode,
       optionalItemNumbersReads
     ).tupled.map((EquipmentDomain.apply _).tupled).map(_(incidentIndex, equipmentIndex))
 
-    lazy val readsWithOptionalContainerId =
+    lazy val readsWithOptionalContainerId: Kleisli[EitherType, UserAnswers, EquipmentDomain] =
```

Comment:
`Kleisli[EitherType, UserAnswers, EquipmentDomain]` is the same as `UserAnswersReader[EquipmentDomain]`